packages/react-core/src/components/TreeView/TreeViewListItem.tsx

```diff
@@ -208,7 +208,6 @@ const TreeViewListItemBase: React.FunctionComponent<TreeViewListItemProps> = ({
             <Component
               className={css(
                 styles.treeViewNode,
-                children && (isSelectable || hasCheckbox) && 'pf-m-selectable',
```

Comment:
Might be worth keeping this in as a string, similar to what was done to [Calendar month in the core bump PR](https://github.com/patternfly/patternfly-react/pull/10081/files#diff-d158e2d7f9e30b2ef3b1527ae50fae669b00830d5d5cbec6c1a74fd1e0721b86R262). Worth getting feedback from core on this, though.

Reply:
@mcoker said to rip it out